python/tests/plugins/lsp_server/definition_constants.py

```diff
@@ -7,12 +8,20 @@
     create_model_definition,
     create_behavior_entry,
     create_scenario_entry,
+    create_schema_ext_definition,
+    _create_parsed_definition
 )
 
 TEST_ENUM = create_enum_definition("Options", ["one", "two", "three"])
 TEST_SCHEMA_A = create_schema_definition("DataA", fields=[create_field_entry("msg", "string")])
 TEST_SCHEMA_B = create_schema_definition("DataB", fields=[create_field_entry("msg", "string")])
 TEST_SCHEMA_C = create_schema_definition("DataC", fields=[create_field_entry("msg", "string")])
+TEST_ROOT_SCHEMA = create_schema_definition("NewRootKeyStructure", fields=[create_field_entry("name", "string"), create_field_entry("test_enum", TEST_ENUM.name)])
+TEST_ROOT_EXTENSION = create_schema_ext_definition(
+    "TestRootExtension", DEFINITION_NAME_ROOT, fields=[create_field_entry("test_root", TEST_ROOT_SCHEMA.name)]
+)
```

Comment:
Formatting is off - `TEST_ROOT_SCHEMA` is all on one line and is longer than `TEST_ROOT_EXTENSION`.

```suggestion
TEST_ROOT_SCHEMA = create_schema_definition("NewRootKeyStructure", fields=[create_field_entry("name", "string"), create_field_entry("test_enum", TEST_ENUM.name)])
TEST_ROOT_EXTENSION = create_schema_ext_definition("TestRootExtension", DEFINITION_NAME_ROOT, fields=[create_field_entry("test_root", TEST_ROOT_SCHEMA.name)])
```